### PR TITLE
Rename timeout to arm cycles

### PR DESCRIPTION
**Pull request by petergaemers**
Rename the references from timouts to arm cycles, since thats what we're tracking

### CODE REVIEW

File: dispatcher/config.ini [hunk 1]
```diff
@@ -36,7 +36,7 @@ RetryReset = 3
 # Min time between Stop-Arm and Arm-Start commands
 TimeBetweenCommands = 6
 #Maximum number of allowed timeouts during the arming sequence before doing something
-MaxTimeout=3
+MaxArmCycles=3
```
Comment (darrylmasson) [inline dispatcher/config.ini:39]: This should also get added to the TESTING block

**Review by darrylmasson — CHANGES_REQUESTED** [review]

File: dispatcher/config.ini [hunk 2]
```diff
@@ -35,8 +35,6 @@ StopCommandTimeout = 30
 RetryReset = 3
 # Min time between Stop-Arm and Arm-Start commands
 TimeBetweenCommands = 6
-#Maximum number of allowed timeouts during the arming sequence before doing something
-MaxTimeout=3
```
[the remark this answers is not in the document]
Comment (darrylmasson) [reply dispatcher/config.ini:39]: > This should also get added to the TESTING block

The key word being "also"

[the remark this answers is not in the document]
Comment (petergaemers) [reply dispatcher/config.ini:39]: Details are not my friends...

**Review by darrylmasson — APPROVED** [review]
I like it

**Pull request merged**